x-pack/test/functional/apps/index_management/feature_controls/index_management_security.ts

```diff
@@ -73,7 +73,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
           // The index_management_user has been given permissions to advanced settings for Stack Management Tests.
           // https://github.com/elastic/kibana/pull/113078/
-          expect(sections).to.have.length(2);
+          expect(sections).to.have.length(3);
```

Comment:
Previously this wasn't seeing the watcher management app since it didn't have direct index access. It DID have `monitor_watcher` access which is included in `monitor` access. We should be relying on the ES privilege instead of the index priv, hence this change.